locations/spiders/lidl_de.py

```diff
@@ -18,67 +13,67 @@
-    start_urls = ["https://www.lidl.de/de/filialsuche/s940"]
+    start_urls = ["https://www.lidl.de/f/"]
 
-    def parse_hours(self, store_hours):
+    def parse_hours(self, hours):
         opening_hours = OpeningHours()
-        if store_hours is None:
-            return
 
-        for store_day in store_hours:
-            day, hours = store_day.split(" ")
-            match = re.match(r"(\d\d:\d\d)-(\d\d:\d\d)", hours)
-            if match:
+        for item in hours:
+            if (item.split()):
+                day = DAY_MAPPING[item.split()[0]]
+                hour = item.split()[1]
                 opening_hours.add_range(
-                    day=DAY_MAPPING[day],
-                    open_time=match.group(1),
-                    close_time=match.group(2),
-                    time_format="%H:%M",
+                    day=day,
+                    open_time=hour.split('-')[0],
+                    close_time=hour.split('-')[1]
                 )
 
         return opening_hours.as_opening_hours()
-
     def parse_details(self, response):
-        properties = {}
-        if response.status == 200:
-            json_data = re.search(r"salePoints = eval\((.*?)\);", response.text)
-            if json_data:
-                json_data = json.loads(json_data.groups(1)[0])
-                for store in json_data:
-                    if store["country"] == "DE":
-                        properties = {
-                            "country": store["country"],
-                            "ref": store["ID"],
-                            "name": store["name"],
-                            "street": store["STREET"],
-                            "postcode": store["ZIPCODE"],
-                            "city": store["CITY"],
-                            "lat": store["X"],
-                            "lon": store["Y"],
-                        }
-                        hours = self.parse_hours(store["parsedopeningTimes"]["regular"])
-                        if hours:
-                            properties["opening_hours"] = hours
-
-                        yield GeojsonPointItem(**properties)
+
+        lidlShops = response.css('.ret-o-store-detail')
+
+        for shop in lidlShops:
+            shopAddress = shop.css('.ret-o-store-detail__address::text').extract()
+            street = shopAddress[0]
+            postalCode = shopAddress[1].split()[0]
+            city = shopAddress[1].split()[1]
+            openingHours = shop.css('.ret-o-store-detail__opening-hours::text').extract()
+            services = response.css('.ret-o-store-detail__store-icon-wrapper')[0]
+            link = services.css('a::attr("href")').get()
+            coordinates = link.split('pos.')[1].split('_L')[0]
+            latitude = coordinates.split('_')[0]
+            longitude = coordinates.split('_')[1]
+
+            properties = {
+                "ref": latitude+longitude,
+                "country": "DE",
+                "street": street,
```

Comment:
```suggestion
                "street_address": street,
```

I don't know the details of German addresses, but this looks like a street address (street name + smaller addr e.g. house number)